### PR TITLE
fix Codecademy CSS/HTML course link broken, update for freeCodeCamp link (resolves #1572)

### LINKED ISSUE #1572: Codecademy CSS/HTML course link broken (no longer free)
### Issue description

Unfortunately codecademy's css/html and web track course in general are no longer free (but the link is still in CSS part of tutorial, leading to a broken url). I can recommend [free code camp's](https://learn.freecodecamp.org/) Responsive Web Design Certification as a reference instead. It has 300 hours of free videos on html/CSS stuff and it's all really good :D 

### Language

English

### Operating system

All

### PR DESCRIPTION
**Pull request by fernandasj**
The css section of the English tutorial had an external link to codeacademy, but it is no longer free. To fix the bug, I modified the link to freeCodeCamp which has a free HTML and CSS course.

### CODE REVIEW

File: en/css/README.md [hunk 1]
```diff
@@ -297,7 +297,7 @@ Look at the code we just pasted to find the places where we added classes in the
 
 Don't be afraid to tinker with this CSS a little bit and try to change some things. Playing with the CSS can help you understand what the different things are doing. If you break something, don't worry – you can always undo it!
 
-We really recommend taking this free online [Codeacademy HTML & CSS course](https://www.codecademy.com/tracks/web). It can help you learn all about making your websites prettier with CSS.
+We really recommend taking this free online [freeCodeCamp - Basic HTML & HTML5](https://learn.freecodecamp.org/). It can help you learn all about making your websites prettier with CSS.
```
Comment (magul) [inline en/css/README.md]: What is free and online? I'm not a native English speaker but would assume, that you loose the `course` word.

Comment (das-g) [inline en/css/README.md]: Is it true that the "Basic HTML and HTML5" course subsection "can help you learn all about making your websites prettier with CSS."? https://learn.freecodecamp.org/ seems to have a separate section (also part of the "Responsive Web Design Certification") about CSS. Should we recommend both subsections? Or even the whole "Responsive Web Design Certification" course?

Comment (das-g) [reply en/css/README.md]: Yeah, it looks like that word got lost.

Comment (fernandasj) [reply en/css/README.md]: sorry, do i need to add the word and resubmit?

Comment (magul) [reply en/css/README.md]: @fernandasj to merge your changes here they need to be accepted. So you need to resolve whatever issue is here, commit that and push it to the same branch. Your updates will occur here automatically.

Comment (magul) [reply en/css/README.md]: Great! Can you also answer a question posted below by @das-g?

Comment (fernandasj) [reply en/css/README.md]: @das-g  I believe the core courses would be ideal for beginners and the topics covered in the section, but we can point to the Responsive Web Design Certification course for further study. Should I put this in README?

Comment (das-g) [reply en/css/README.md]: Which one(s) is / are "core courses"?

Comment (fernandasj) [reply en/css/README.md]: the main topics covered in the session are HTML and CSS, freeCodeCamp has the general course from the most advanced basics in Web Design

Comment (das-g) [reply en/css/README.md]: Not quite sure I understand. How about mentioning both their HTML course and their CSS course? E.g. like this:

```suggestion
We really recommend taking the free online courses "Basic HTML & HTML5" and "Basic CSS" on [freeCodeCamp](https://learn.freecodecamp.org/). They can help you learn all about making your websites prettier with HTML and CSS.
```

File: en/css/README.md [hunk 2]
```diff
@@ -297,7 +297,7 @@ Look at the code we just pasted to find the places where we added classes in the
 
 Don't be afraid to tinker with this CSS a little bit and try to change some things. Playing with the CSS can help you understand what the different things are doing. If you break something, don't worry – you can always undo it!
 
-We really recommend taking this free online [Codeacademy HTML & CSS course](https://www.codecademy.com/tracks/web). It can help you learn all about making your websites prettier with CSS.
+We really recommend taking this free online [freeCodeCamp - Basic HTML & HTML5 course](https://learn.freecodecamp.org/). It can help you learn all about making your websites prettier with CSS.
```
Comment (das-g) [inline en/css/README.md]: My suggestion based on the current diff:

```suggestion
We really recommend taking the free online courses "Basic HTML & HTML5" and "Basic CSS" on [freeCodeCamp](https://learn.freecodecamp.org/). They can help you learn all about making your websites prettier with HTML and CSS.
```

**Comment by fernandasj** [comment]
Awesome thank you :tada:

**Pull request merged**